DigitalLearningSolutions.Data.Migrations/202405031044_AlterGetCurrentCoursesForCandidate_V2.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202405151044)]
+    public class AlterGetCurrentCoursesForCandidate_V2 : Migration
+    {
+        public override void Up()
+        {
+            Execute.Sql(Properties.Resources.TD_3671_Alter_GetCurrentCoursesForCandidate_V2_proc1_up );
+            Execute.Sql(Properties.Resources.TD_3671_Alter_CheckDelegateStatusForCustomisation_func_up );
+        }
+        public override void Down()
+        {
+            Execute.Sql(Properties.Resources.TD_3671_Alter_GetCurrentCoursesForCandidate_V2_proc_down);
```

Comment:
There are extra brackets in the down queries. The ones in SSMS and written here do not exactly match. Please see this again.